scripts/build_biomass_potentials.py

```diff
@@ -28,7 +28,8 @@ def build_nuts_population_data(year=2013):
     swiss = [swiss.groupby(swiss.index.str[:i]).sum() for i in range(2, 6)]
 
     # merge Europe + Switzerland
-    pop = pd.DataFrame(pop.append(swiss), columns=["total"])
+    pop = pd.concat([pop, pd.concat(swiss)])
+    pop = pd.DataFrame(pop, columns=["total"])
```

Comment:
can we move the inner concat to the line above?

so that we can write
```suggestion
    pop = pd.concat([pop, swiss]).to_frame("total")
```
instead

Reply:
Maybe something in the middle? swiss is a list, while pop was a panda series. I've added the changes